packages/ui/v2/core/MeetingTimeInTimezones.tsx

```diff
@@ -54,14 +54,14 @@ const MeetingTimeInTimezones = ({
     <Popover.Root>
       <Popover.Trigger
         onClick={preventBubbling}
-        className="popover-button ml-2 inline-flex h-5 w-5 items-center justify-center rounded-sm text-gray-900 transition-colors hover:bg-gray-200 focus:bg-gray-200">
+        className="popover-button invisible ml-2 inline-flex h-5 w-5 items-center justify-center rounded-sm text-gray-900 transition-colors hover:bg-gray-200 focus:bg-gray-200 group-hover:visible">
```

Comment:
i dont think we will ever use the MeetingTimeInTimezone elsewhere but if that is the case, we should move the "invisible" into a parent container outside of this component